var/spack/repos/builtin/packages/py-morphio/package.py

```diff
@@ -17,6 +17,7 @@ class PyMorphio(PythonPackage):
     version('unifurcation', branch='unifurcation', submodules=True, get_full_repo=True)
 
     version('3.1.1', sha256="ad9f0e363f09f03c6eda54f5f3b006d204236677d2f2c9675421e0441033a503")
+    version('2.7.1', sha256="3f3e2229da85e874527775fce080f712b6dc287edc44b90b6de35d17b34badff")
```

Comment:
Shouldn't we update also the spack package for `morphio` when `py-morphio` is updated?

Reply:
thank you! I didn't know that the package is duplicated. Why there are `morphio` and `py-morphio`?

Reply:
Because it's hard to build both at the same time :shrug: